bench/lib/Helpers.re

```diff
@@ -25,6 +25,10 @@ let simpleState = {
 
 let defaultFont: Service_Font.font = {
   fontFile: Revery.Environment.executingDirectory ++ "FiraCode-Regular.ttf",
+  fontFamily:
+    Revery.Font.Family.fromFile(
+      Revery.Environment.executingDirectory ++ "FiraCode-Regular.ttf",
+    ),
```

Comment:
I actually don't think you need to prepend the executing directory. I originally had that function named `Family.asset` but @bryphe (rightfully) recommended I change it to something a little more obvious that it could be from any file.

The file string eventually gets passed here:
https://github.com/revery-ui/revery/blob/c4c6c23f97a0278e1877d37306bdb89e45bb439e/src/Font/FontCache.re#L66

Which, as you can see, calculates the path as an asset anyway. I can check if you want, but it might be easier for you since you already have it built. Let me know!

Reply:
Yeah I wondered about that too, as it's not done anywhere else. Hard to verify here though since this is in the benchmark, although I suppose it doesn't really matter as long as some font is loaded.